tests/test_simulation.py

```diff
@@ -887,7 +887,7 @@ def test_noises_all(matrices, noise, result, n_collapse_ops, seq):
             noise=noise,
             dephasing_rate=0.1,
             hyperfine_dephasing_rate=0.1,
-            relaxation_rate=1000,
+            relaxation_rate=2000,
```

Comment:
It's slightly ridiculous that we have to go this high in relaxation rate to see an effect. I would say it is preferable to change the sequence to something where relaxation has an effect, even if just for this one case - otherwise we're not really testing anything